llm/mcloud/mcli-1b-custom.yaml

```diff
@@ -26,19 +26,20 @@ cluster: r0z0 # replace with your cluster here!
 # and referenced in our `command` above
 # We use this to customize hparams like `max_seq_len=8192` at runtime
 parameters:
-  data_remote: &data_remote ./my-copy-c4
   data_local: &data_local ./my-copy-c4
-  max_seq_len: &max_seq_len 8192
```

Comment:
Can you explain these changes in the PR?

Reply:
Ah, I think as I was trying to update it, I copied over the 125M model and did not preserve some of the custom flags. Fixing it now!